open-craft/opencraft
# [BB-3500] Adding CoursesManage component and sidebar integration

**Summary:**  We're fulfilling the following user story:

_"As a OCIM user, I want to have dedicated page to point me out to studio link so that I avoid confusion"_

We're adding a new category, in the form of yet anohter accordion component, in the customization side menu called Courses (Studio) which holds the following page: (View screenshot)


**JIRA tickets**: [[BB-3500]](https://tasks.opencraft.com/browse/BB-3500)

**Dependencies**: None

**Screenshots**: ![image](https://user-images.githubusercontent.com/33784039/106534501-991e4000-64ca-11eb-9a12-644a2b053b8a.png)

**Testing instructions**:

1. Pull `danielf/BB-3500` and do the necessary changes to have the frontend app run (my suggestion on how to do this is on the mentioned ticket).
2. Click the accordion showcased in the screenshot. Verify that the links work (this is also implemented in the tests)

**Author notes and concerns**:
- The Zepplin mockups are not 100% true to the current OCIM design. Some minimal liberties were taken, please tell if there's anything that doesn't look right.
- According to [this](https://handbook.opencraft.com/en/latest/coding_standards/), we just "use TypesScript," so I'm trusting `npm lint`. Please tell if there are more considerations. 
- I rebuilt package-json.lock. Any comments on this? 

**Reviewers**
